test/integration/serverless/snapshots/timeout-csharp

```diff
@@ -1 +1,3 @@
-
+{
+  "error": "normalization returned empty payload"
+}
```

Comment:
This is the error message returned by the python normalizer script. By storing it here, we can actually run all of our tests (skipping zero of them).